Gemfile

```diff
@@ -118,7 +118,7 @@ platforms :ruby, :mswin, :mswin64, :mingw, :x64_mingw do
   gem "racc", ">=1.4.6", require: false
 
   # Active Record.
-  gem "sqlite3", "~> 1.3.6"
+  gem "sqlite3", "~> 1.3", ">= 1.3.6"
 
```

Comment:
I **really** think you want this to be `">= 1.3.6", "< 2.0"`. When/if sqlite3 releases a backwards-incompat 2.0, it's going to cause a whole lot of confusion when it's specified like this.